server/testutil/auth/auth.go

```diff
@@ -7,11 +7,12 @@ import (
 
 	"github.com/buildbuddy-io/buildbuddy/server/interfaces"
 	"github.com/buildbuddy-io/buildbuddy/server/tables"
+	"github.com/buildbuddy-io/buildbuddy/server/util/capabilities"
 	"github.com/buildbuddy-io/buildbuddy/server/util/status"
 
 	"google.golang.org/grpc/metadata"
 
-	akpb "github.com/buildbuddy-io/buildbuddy/proto/api_key"
+	"github.com/buildbuddy-io/buildbuddy/enterprise/server/auth"
```

Comment:
it strikes me as a little error-prone to import this into a package with the same name. Can we call it realauth or something?

Reply:
Done